### DISCUSSION TITLE
IOTA lottery

### DISCUSSION THREAD

**Discussion by ashnik** [iotaledger/firefly#8211] [Help & Support]
I get a message in Firefly......won  some IOTA in lottery .......click to claim and validate your gain at iotalottery.org.
There is no place to click to claim.
Can anybody help ?

**Comment by MiKeF05** [comment]
That´s a scam. Do not enter any information or your seed.

**Comment by ashnik** [comment]
Thks

**Comment by begonaalvarezd** [comment]
Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
1) In your wallet go to: Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
4) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/5ecce50b-92af-4a09-97e8-5f72cf1bf9b8)

**Comment by jambtc** [comment]
WARNING! The scam site is changed in iotafirefly dot com

**Comment by KmdKeen** [comment]
Got this stuff in my wallet as well, but no option to burn this stuff:

<img width="371" height="704" alt="image" src="https://github.com/user-attachments/assets/20f6123d-0a9d-4159-b912-bfdd20309eef" />

Is there anything else I can do to get rid of this?
